src/containers/PeopleList.jsx

```diff
@@ -129,21 +129,19 @@ export class PeopleList extends Component {
         }
       }
     ];
-    if (window.PASSPORT_STRATEGY !== "slack") {
```

Comment:
Why is this if statement being removed?

Reply:
It must've gotten lost in the refactoring.  Great catch Kathy!